warp/yul/ExpressionSplitter.py

```diff
@@ -40,6 +40,19 @@ def visit_function_call(self, node: ast.FunctionCall):
         var_name = self.env.process_subexpr(node)
         return ast.Identifier(var_name)
 
+    def visit_literal(self, node: ast.Literal):
+        if len(self.path) == 1:
+            assert (
+                False
+            ), "Literals shouldn't be at top level, these should be optimised away at this point"
+        parent = self.path[-2]
+        if isinstance(
+            parent, (ast.Assignment, ast.VariableDeclaration, ast.ExpressionStatement)
+        ):
+            return node
+        var_name = self.env.process_subexpr(node)
+        return ast.Identifier(var_name)
+
```

Comment:
Can I see this change?

Reply:
https://github.com/NethermindEth/warp/pull/122/commits/8fff3df8b1e14631db5358559bc5309805051eaf

Reply:
Well, not quite. It can happen in `if-expressions` because we do `if x.low + x.high != 0`, I guess. If this doesn't come up with this PR though, let's not worry about it. It's better fixed in ToCairoVisitor, by simply checking if the condition is a literal

Reply:
Fine, I'll remove it

Reply:
It didn't work for me to just remove it, there was some error about top level uninstantiated variables in `return-var-capturing` and only there. I didn't see any variables that fit the descriptiong in that contract though